src/Compatibility/Core/src/iOS/Renderers/OpenGLViewRenderer.cs

```diff
@@ -90,7 +95,7 @@ void SetupRenderLoop(bool oneShot)
 			_displayLink.AddToRunLoop(NSRunLoop.Current, NSRunLoop.NSRunLoopCommonModes);
 		}
 
-		class Delegate : GLKViewDelegate
+		class Delegate : GLKViewDelegate, IGLKViewDelegate
```

Comment:
Why is this needed? Doesn't `GLKViewDelegate` already implement `IGLKViewDelegate`?

Reply:
It was complaining about not being able to implicitly cast and this made the compiler happy